beacon-chain/rpc/apimiddleware/structs.go

```diff
@@ -118,30 +118,6 @@ type LightClientUpdateDataJson struct {
 	SignatureSlot           string                 `json:"signature_slot"`
 }
 
-type LightClientFinalityUpdateResponseJson struct {
```

Comment:
Since we wish to remove below, then for above full version of json update, we might need to add omitempty

Reply:
ah good catch